dist/api_data.py

```diff
@@ -1573,6 +1573,10 @@ def __ge__(self, other):
         Display page addresses, time windows, and page types as pages are verified, using the
         application's message handler, intended for debugging''',
         type='boolean'),
+    Config('dump_app_data', 'true', r'''
+        Display application data as pages or blocks are verified,
+        using the application's message handler, intended for debugging.''',
```

Comment:
I think this deserves an additional sentence. "Disabling this does not guarantee that no user data will be output". I think that's worthwhile, since I don't want to incur the audit overhead of guaranteeing user data is redacted.

Reply:
Also do you want it to default to 'true'? For logging redacting log record content is the default and the user (like the wt command) has to "unredact" it explicitly.

Reply:
@agorrod done!

@sueloverso I am indifferent on what the default behaviour on this since I see benefits and disadvantages on both sides. @agorrod what are your thoughts?